### ISSUE TITLE
qiskit-aer and qiskit-terra

### ISSUE THREAD

**Issue by JanLahmann** [piwheels/packages#198]
For qiskit-aer and qiskit-terra the latest versions are missing (aer=0.8.0 and tera=0.17.0).

Package name: qiskit-aer and qiskit-terra
Link to PyPI page: https://pypi.org/project/qiskit-terra/ and https://pypi.org/project/qiskit-aer
Link to piwheels page: https://www.piwheels.org/project/qiskit-terra/ and https://www.piwheels.org/project/qiskit-aer/
Version: aer=0.8.0 and tera=0.17.0
Python version: 3.7.3
I am the maintainer: No
More information:

Qiskit-terra will be most probably be solved with https://github.com/Qiskit/qiskit-terra/pull/6163 

For qiskit-aer, the following procedure works on my local system:

```bash
pip3 install cmake
sudo pip3 install conan
pip3 install qiskit-aer
```

Is this possible on the piwheels build system as well?

Thanks, Jan

**Comment by mtreinish** [comment]
The qiskit-terra failure will be fixed by https://github.com/Qiskit/qiskit-terra/pull/6163 it was an oversight in the packaging for the release (the sdist is broken). There will be a 0.17.1 release which will fix it.

**Comment by JanLahmann** [comment]
Qiskit-terra is now fixed with the 0.17.1 release. Thank you, @mtreinish !

For qiskit-aer, the following build procedure works on a clean Raspbian OS:
```
sudo apt -y install libatlas-base-dev
sudo pip3 install cmake scipy
sudo pip3 install qiskit-aer
```
Is this feasible on the piwheels.org build system?

**Comment by bennuttall** [comment]
atlas, cmake and scipy are already available on piwheels builders.

terra has built successfully, but aer just ran out of memory: http://paste.debian.net/1194410/

**Comment by JanLahmann** [comment]
@bennuttall , thank you for the info!
I'm building on Pi4 with 2GB RAM :-)

**Comment by bennuttall** [comment]
I'll try myself when I get chance - it might work if we try again

**Comment by JanLahmann** [comment]
did you have a chance to try this again, yet?

**Comment by bennuttall** [comment]
Just managed to build it on an 8GB Pi 4. I'll kick off the build again to see if it works.

**Comment by bennuttall** [comment]
No, it ran out of memory again. I'll import my build for the latest version. Are there likely to be regular releases?

**Comment by JanLahmann** [comment]
Thank you! This is great !!
The complete Qiskit install with all modules now completes in less than 15 minutes on a RPi 4. Install script is available as part of the RasQberry project, http://rasqberry.org
(Just llvmlite needs to be compiled locally.)
